aim/web/ui/src/services/models/explorer/createAppModel.ts

```diff
@@ -2294,6 +2282,28 @@ function createAppModel(appConfig: IAppInitialConfig) {
         });
       }
 
+      function updateTableData(
+        tableData: {
+          rows: any;
+          sameValueColumns: string[];
+        },
+        tableColumns: ITableColumn[],
+        configData: IAppModelConfig | any,
+      ): void {
+        if (updateTableTimeoutId) {
+          clearTimeout(updateTableTimeoutId);
+        }
+
+        updateTableTimeoutId = setTimeout(() => {
```

Comment:
here let's use `window.setTimeout` and fix the type of `updateTableTimeoutId: ReturnType<typeof setTimeout>;`
to number

Reply:
@KaroMourad  There are no difference, except that windows.setTimeout works only in browser environment

Reply:
@arsengit we can use it only on browser env that's why we can use `window.setTimeout` here